src/tactic/ui/startup/column_edit_wdg.py

```diff
@@ -103,7 +103,9 @@ def get_display(my):
                     names.push(name);
                 }
 
-                spt.table.add_columns(names)
+                // Unless there is a table here, we should not do this.
+                // Better handled with a callback
+                //spt.table.add_columns(names)
```

Comment:
so it's gonna add a column without showing it in case tablelayout is in the view?